integration/build.gradle

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
This would be better if it were done the same way as integration tests in core, so that it can all be run in one command line, eg
```
./gradlew clean check integrationTest
```
...rather than being a separate project.